src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.PackageDependencyResolution.targets

```diff
@@ -261,6 +261,37 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <!--
     ============================================================
+    ProjectReference Targets: Include transitive project references before 
+                              ResolveProjectReferences is called
+    - _ComputeTransitiveProjectReferences
+    - IncludeTransitiveProjectReferences
+    ============================================================
+    -->
+
+  <Target Name="_ComputeTransitiveProjectReferences"
+          DependsOnTargets="RunResolvePackageDependencies"
+          Returns="_TransitiveProjectReferences">
+    <ItemGroup>
+      <ProjectDefs Include="@(PackageDefinitions->WithMetadataValue('Type', 'project'))" />
```

Comment:
Nit: Let's be consistent on not abbreviating things. We have PackageDefinitions and ProjectDefs on the same line here. It should be ProjectDefinitions.

Item names are also supposed to be singular, but that's one Microsoft has gotten wrong in many places so I feel less strongly about enforcing it.